Converters/CSi/Speckle.Converters.ETABSShared/ToSpeckle/Helpers/ApplicationPropertiesExtractor.cs

```diff
@@ -0,0 +1,57 @@
+using Speckle.Converters.CSiShared;
+using Speckle.Converters.CSiShared.ToSpeckle.Helpers;
+
+namespace Speckle.Converters.ETABSShared.ToSpeckle.Helpers;
+
+/// <summary>
+/// ETABS-specific property extractor that composes with SharedPropertiesExtractor to provide
+/// both shared and ETABS-specific properties.
+/// </summary>
+/// <remarks>
+/// Follows the composition pattern where SharedPropertiesExtractor handles common CSI properties,
+/// while this class adds ETABS-specific properties. The extraction order is important:
+/// 1. Extract shared properties first via SharedPropertiesExtractor
+/// 2. Augment with ETABS-specific properties
+/// This ensures consistent base properties with ETABS-specific enrichment.
+/// </remarks>
+public class ApplicationPropertiesExtractor : IApplicationPropertiesExtractor
```

Comment:
Small thing, but calling this class `EtabsPropertiesExtractor: IApplicationPropertiesExtractor` would be more descriptive

Reply:
Done 👍🏼